src/assets/contributors-map.geojson

```diff
@@ -30,6 +30,21 @@
           39.659768
         ]
       }
-    }
+    },
+    {
+      "type": "Feature",
+      "properties": {
+        "firstName": "Bader",
+        "githubId": "baderk",
+        "favoriteCrag": "becfb41f-3278-59c6-a376-2d23d97c220d"
+      },
+      "geometry": {
+        "type": "Point",
+        "coordinates": [
+          36.037803,
```

Comment:
can you check if the coordinates are in [longitude, latitude] order?
Geojson specification: https://www.rfc-editor.org/rfc/rfc7946#section-3.1.1

If you can also use https://geojson.io/ to check the format of your file.  You should see drop pins if the format is correct.

Reply:
Thanks for pointing out the specification and the geojson tool. I just addressed it and I see the 3 points now on the map 👍🏼